packages/connector/src/services/accounts/tests/helpers/asset.ts

```diff
@@ -0,0 +1,14 @@
+import { randomInt } from 'crypto'
+
+// Use unique assets as a workaround for not being able to reset
+// Tigerbeetle between tests
```

Comment:
What's the blocker on resetting TigerBeetle? (or will testcontainers work resolve this?)